source/config.cpp

```diff
@@ -364,6 +370,19 @@ bool Config::is_class_skipping_requested(string const &class__) const
 }
 
 
+bool Config::is_field_skipping_requested(string const &field_) const
+{
+	string field {field_};
+	field.erase(std::remove(field.begin(), field.end(), ' '), field.end());
```

Comment:
Are you sure this is needed? At line 214 https://github.com/RosettaCommons/binder/pull/257/files#diff-72f8d59ab463236c54ef8c3a2c8fb31c72efad5003751b3e342c868079de19baR214 we added fields name without namespaces so it looks like it this line will not do anything. I am brining this up due to performance consideration: in large projects this check might be called numerous times.